tests/OpenTelemetry/AWSXRay/TraceInfoSpec.hs

```diff
@@ -0,0 +1,90 @@
+module OpenTelemetry.AWSXRay.TraceInfoSpec
```

Comment:
Are these the specs we ported? Should we link somewhere here too just for coherency/consistency?

Reply:
No, I wrote them fresh. The only stuff I "ported" was the `Baggage` de/encoding (and they weren't really ported so much as just written to represent similar cases), and the comment about that includes a link.

Reply:
Gotcha, makes sense!